eZ/Bundle/EzPublishCoreBundle/Command/CheckURLsCommand.php

```diff
@@ -45,18 +74,14 @@ public function configure()
         );
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): void
     {
-        $repository = $this->getContainer()->get('ezpublish.api.repository');
-        $repository->getPermissionResolver()->setCurrentUserReference(
-            $repository->getUserService()->loadUserByLogin($input->getOption('user'))
+        $this->permissionResolver->setCurrentUserReference(
+            $this->userService->loadUserByLogin($input->getOption('user'))
```

Comment:
User option is not required which can generate error here

Reply:
If the user option is not specified, `admin` will be used as default.  Ref. 

https://github.com/ezsystems/ezpublish-kernel/blob/5e2f92bf37f53529e55a06e74c3b4b9cf32002ed/eZ/Bundle/EzPublishCoreBundle/Command/CheckURLsCommand.php#L68-L74

Other commend also follow this pattern.

Reply:
right 👍